tests/doctrine/DoctrineCleanInsert1Test.php

```diff
@@ -1,12 +1,7 @@
 <?php
-
-//require_once 'PHPUnit/Extensions/Database/TestCase.php';
-//require_once 'PHPUnit/Extensions/Database/DataSet/DefaultDataSet.php';
 require_once dirname(__FILE__) . '/TestUtil.php';
-
-use Doctrine\ORM\EntityManager;
 require_once dirname(__FILE__) . "/bootstrap.php";
-
+use Doctrine\ORM\EntityManager;
```

Comment:
So this line moved?

Reply:
For neatness I moved it after the requires, partly to check my understanding that the order wouldn't matter. That said, I didn't do it on other files, so am happy to undo this if needed

Reply:
Just wanted to make sure it was intentional!